atlas_placement_hints/app/placement_hints.py

```diff
@@ -305,41 +311,98 @@ def isocortex(
+    load_cut_thalamus_meshes_flag,
 ):
     """Generate and save the placement hints of the mouse thalamus.
 
-    Placement hints are saved under the names sepecified in `app/metadata/thalamus_metadata.json`.
-    Default to:
+    Note that you MUST pass either '--create-uncut-thalamus-meshes-flag' or
+    '--load-cut-thalamus-meshes-flag'.
+
+    Placement hints are saved under the names specified in
+    `app/metadata/thalamus_metadata.json`. These default to:
 
     \b
     - `[PH]y.nrrd`
-    - `[PH]Rt.nrrd`, `[PH]VPL.nrrd`
+    - `[PH]layer_1.nrrd` (This is the "top-most" layer, equivalent to "RT".
+      This previously used the filename `[PH]RT.nrrd`)
+    - `[PH]layer_2.nrrd` (This is the "deepest" layer, equivalent to the
+      thalamus except the habenular, peripeduncular, and reticular regions.
+      This previously used the filename `[PH]THnotRT.nrrd`)
 
-    A report together with an nrrd volume on problematic distance computations are generated
-    in `output_dir` under the names:
+    A report together with an nrrd volume on problematic distance computations
+    are generated in `output_dir` under the names:
 
     \b
     - `distance_report.json`
-    - `<Thalamus>_problematic_voxel_mask.nrrd` (mask of the voxels for which the computed
-    placement hints cannot be trusted).  <Thalamus> is the region name specified in
-    thalamus_metadata.json. Defaults to "Thalamus".
-
-    The annotation file can contain the thalamus or a superset.
-    For the algorithm to work properly, some space should separate the boundary
-    of the thalamus from the boundary of its enclosing array.
+    - `<Thalamus>_problematic_voxel_mask.nrrd` (mask of the voxels for which
+      the computed placement hints cannot be trusted).  <Thalamus> is the
+      region name specified in thalamus_metadata.json. Defaults to "Thalamus".
+
+    The annotation file can contain the thalamus or a superset. For the
+    algorithm to work properly, some space should separate the boundary of the
+    thalamus from the boundary of its enclosing array.
+
+    For instructions on all steps necessary to generate the thalamus' placement
+    hints, see
+    'atlas-placement-hints/atlas_placement_hints/layered_atlas.py:ThalamusAtlas'
+    and its methods for details.
     """
     set_verbose(L, verbose)
 
     atlas = _create_layered_atlas(annotation_path, hierarchy_path, metadata_path)
-    _placement_hints(
-        atlas,
-        direction_vectors_path,
-        output_dir,
-        has_hemispheres=True,
-    )
+
+    if create_uncut_thalamus_meshes_flag:
+        Path(thalamus_meshes_dir).mkdir(parents=True, exist_ok=True)
+        atlas.create_uncut_thalamus_meshes(thalamus_meshes_dir)
+    elif load_cut_thalamus_meshes_flag:
+        _placement_hints(
+            atlas,
+            direction_vectors_path,
+            output_dir,
+            thalamus_meshes_dir=thalamus_meshes_dir,
+            has_hemispheres=True,
+        )
+    else:
+        print(
```

Comment:
don't print in these cases, raise an Exception as the code will not work, its better to raise a clear exception, the user may not read the log properly, and get into another bug, harder to understand